src/app/modules/dashboard/dashboard-view/dashboard-view.component.html

```diff
@@ -1,14 +1,49 @@
-<div>
+<div id="dashboard">
     <div class="breadcrumb">
         <div class="breadcrumb-titles">
             <mat-icon class="icon-cls">home</mat-icon>
             <span class="navigationcls">Grievance Management</span>
             <span class="navigationcls active">Dashboard</span>
         </div>
     </div>
-    <div>
+    <div class="inner-content">
         <app-shared-skeleton-loading [isLoading]="isDataLoading">
         </app-shared-skeleton-loading>
+        <div *ngIf="!isDataLoading">
+            <mat-form-field>
+                <mat-label>Enter a date range</mat-label>
+                <mat-date-range-input [rangePicker]="picker">
+                  <input matStartDate placeholder="Start date">
+                  <input matEndDate placeholder="End date">
+                </mat-date-range-input>
+                <mat-hint>MM/DD/YYYY – MM/DD/YYYY</mat-hint>
+                <mat-datepicker-toggle matIconSuffix [for]="picker"></mat-datepicker-toggle>
+                <mat-date-range-picker #picker></mat-date-range-picker>
+              </mat-form-field>
+        </div>
+        <br/>
+        <br/>
+        <div *ngIf="!isDataLoading">
+        <mat-grid-list cols="6" rowHeight="10rem" [gutterSize]="'100px'">
+            <mat-grid-tile colspan="4" rowspan="1">
+                <app-shared-table class="table1" [hasFilterOptions]="false" [tableColumns]="dashboardAnaytics1Columns" [tableData]="dashboardAnalytics1Data" [isPageable]="false"></app-shared-table>
+            </mat-grid-tile>
+            <mat-grid-tile colspan="2" rowspan="1">
+                <app-shared-table class="table2" [hasFilterOptions]="false" [tableColumns]="dashboardAnalytics3Columns" [tableData]="dashboardAnalytics3Data" [isPageable]="false"></app-shared-table>
+            </mat-grid-tile>
+        </mat-grid-list>
+          <mat-grid-list cols="6" rowHeight="10rem" [gutterSize]="'100px'">
+            <mat-grid-tile colspan="4" rowspan="1">
+                <app-shared-table class="table1" [hasFilterOptions]="false" [tableColumns]="dashboardAnalyticsColumns" [tableData]="dashboardAnalyticsData" [isPageable]="false"></app-shared-table>
+            </mat-grid-tile>
+            <mat-grid-tile colspan="2" rowspan="1">
+                <app-shared-table class="table2" [hasFilterOptions]="false" [tableColumns]="dashboardAnaytics2Columns" [tableData]="dashboardAnalytics2Data" [isPageable]="false"></app-shared-table>
+            </mat-grid-tile>
+        </mat-grid-list>
+        <br/>
```

Comment:
@RoopashreeLakshmikeshava  Avoid using br tag , rather using margin and padding for spacing

Reply:
Fixing it.